website/docs/guides/markdown-features/markdown-features-admonitions.mdx

```diff
@@ -121,3 +121,35 @@ import TabItem from '@theme/TabItem';
 ```
 
 :::
+
+## Custom Admonitions
+
+If you're using `@docusaurus/preset-classic`, you can create custom admonitions by adding them to your website's `docusaurus.config.js`. 
+
+For example, the following configuration adds an admonition with the `custom` keyword. This admonition will be styled as an [Infima `alert-info` component](https://infima.dev/docs/components/alert):
+
+```js title="docusaurus.config.js"
+module.exports = {
+  // ...
+  presets: [
+    [
+      "@docusaurus/preset-classic",
+      {
+        docs: {
+          // ...
+          admonitions: {
+            customTypes: {
+              custom: {
+                ifmClass: "info",
```

Comment:
Remove infima class

Reply:
This is directly passed to `remark-admonitions`. Surprising it may sound, but `remark-admonitions` is built on top of Infima. I think we should build an unbiased remark plugin using [Remark directive](https://github.com/remarkjs/remark-directive) ourselves 🤷‍♂️